code/script.js

```diff
@@ -231,95 +233,138 @@ const setSecret = () => {
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  generateBoard()
```

Comment:
The selectQuestion function should be up here aswell. You can't press on the find out button if you haven't changed the question. Or did you not want the function to be invoked from the start?